velox/functions/prestosql/ArrayFunctions.h

```diff
@@ -639,4 +639,38 @@ struct ArrayFlattenFunction {
     }
   }
 };
+
+/// This class implements the array remove function.
+///
+/// DEFINITION:
+/// array_remove(x, element) → array
+/// Remove all elements that equal element from array x.
+template <typename TExecCtx, typename T>
+struct ArrayRemoveFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(TExecCtx);
+
+  FOLLY_ALWAYS_INLINE void call(
```

Comment:
no need for FOLLY_ALWAYS_INLINE since inline does not have much value here

Reply:
@laithsakka remove FOLLY_ALWAYS_INLINE

Done.